shared/src/commonTest/kotlin/me/saket/press/shared/note/FakeNoteRepository.kt

```diff
@@ -29,23 +33,23 @@ class FakeNoteRepository : NoteRepository {
         includeEmptyNotes -> savedNotes
         else -> savedNotes.filter { it.content.isNotEmpty() }
       }
-    }
+    }.observeOn(scheduler)
```

Comment:
What's the reason for explicitly specifying the schedulers here in fake code when `observableFromFunction()` will run on the thread it was subscribed on anyway?

Reply:
I'm still thinking about it but the reason is to freeze the downstream. Maybe it worth to hide it behind a meaningful extension function.